_data/ui/destination-settings-page.yml

```diff
@@ -50,6 +52,12 @@ default-copy:
     Enter the password associated with the Stitch {{ destination.display_name }} user.
   database: &database-copy |
     Enter the name of the {{ destination.display_name }} database that you created for Stitch.
+  ssl: &ssl-copy |
+    Check to connect to {{ destination.display_name }} using SSL. **Note**: This is not required to connect Stitch.
+  ssl-certificate: &ssl-certificate-copy |
+    **Optional**: Provide the certificate (typically a CA or server certificate) Stitch should verify the SSL connection against. The connection will succeed only if the server's certificate verifies against the certificate provided here.
+
+    **Note**: Providing a certificate isn't required to use SSL. This is only if Stitch should verify the connection against a specific certificate.
```

Comment:
@cmerrick Ok - I added a brief description about the cert box here, based on what you wrote originally. I believe this line is correct, but I'd feel better about the wording if you could verify its accuracy for me :)